From 84c687ce7178f92097929bfff560c470ceefd83a Mon Sep 17 00:00:00 2001
From: Kody Kantor <kody@kkantor.com>
Date: Wed, 28 Nov 2018 21:01:44 +0000
Subject: [PATCH] joyent/pgstatsmon#20 link to in-depth documentation in README
 Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com> Reviewed by:
 David Pacheco <dap@joyent.com> Approved by: David Pacheco <dap@joyent.com>

---
 README.md | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/README.md b/README.md
index aacc88d..421c968 100644
--- a/README.md
+++ b/README.md
@@ -202,6 +202,10 @@ directory:
 * querystat.d reports latency and errors for a given backend's queries
 * walstat.d reports WAL throughput stats for all of pgstatsmon's backends
 
+## Documentation
+
+The [docs](./docs) directory contains documents describing how pgstatsmon works
+in more detail.
 
 ## License
 MPL-v2. See the LICENSE file.
-- 
2.21.0

